Source/WebCore/Modules/applepay/ApplePaySession.cpp

```diff
@@ -67,6 +67,10 @@
 #include <wtf/IsoMallocInlines.h>
 #include <wtf/RunLoop.h>
 
+#if ENABLE(APPLE_PAY_DEFERRED_PAYMENTS)
+# include <wtf/DateMath.h>
+#endif // ENABLE(APPLE_PAY_DEFERRED_PAYMENTS)
```

Comment:
Nit: Trailing comment is not really necessary here, since it's a very short section